### ISSUE TITLE
CDC-Cloud: owner exited with error "code = ResourceExhausted desc = trying to send message larger than max (7749468 vs. 2097152)"

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/tiflow#2190]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- Import workload 6w table count, 550GB data size to upstream TiDB.
- Create a changefeed

2. What did you expect to see?

3. What did you see instead?

- CDC log:

  
![image](https://user-images.githubusercontent.com/78345569/123781416-cef31380-d906-11eb-8ae9-544ba81c7032.png)

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
       5.0.2
      tikv-cloud-cdc-5.0-52f335601-2021-06-26-22-04.tar.gz
        ```

    - TiCDC version (execute `cdc version`):

        ```
        cdc-cloud-cdc-5.0-9300b05c-2021-06-26-22-17.tar.gz
        ```

**Comment by lance6716** [comment]
may take a look at https://github.com/pingcap/dm/pull/1624 @liuzix

**Comment by overvenus** [comment]
This issue can be work around by https://github.com/pingcap/ticdc/pull/2223, and we are developing p2p metadata management. #2596

**Comment by overvenus** [comment]
We have confirmed this issue is fixed in release-5.4.
